docs/website/docs/dlt-ecosystem/verified-sources/rest_api.md

```diff
@@ -0,0 +1,220 @@
+---
+title: REST API generic source
+description: dlt verified source for REST APIs
+keywords: [rest api, restful api]
+---
+import Header from './_source-info-header.md';
+
+# REST API Generic Source
+
+<Header/>
+
+This is a generic dlt source you can use to extract data from any REST API. It uses declarative configuration to define the API endpoints, their relationships, parameters, pagination, and authentication.
+
+## Setup Guide
+
+### Initialize the verified source
+
+Enter the following command:
+
+   ```sh
+   dlt init rest_api duckdb
+   ```
+
+[dlt init](../../reference/command-line-interface) will initialize the pipeline example with REST API as the [source](../../general-usage/source) and [duckdb](../destinations/duckdb.md) as the [destination](../destinations).
```

Comment:
I would also mention here that this command will create pipeline example for github api

Reply:
and pokemon

Reply:
[nit]